docs/installation.rst

```diff
@@ -169,16 +183,34 @@ Running the following command in the python (virtual) environment in Terminal/Po
 
     cd OpenQL
     git submodule update --init --recursive
-    python setup.py install
+    pip install -v
 
 Or in editable mode by the command:
 
 ::
 
-    pip install  -e .[develop]
+    pip install -v -e .
+
+Editable mode has the advantage that you'll get incremental compilation if you ever change OpenQL's C++ files, but it's
+a bit more fragile in that things will break if you move the OpenQL repository around later.
```

Comment:
so when you move the repo, clean first, that's what you mean?

Reply:
Also, about the "having to clean after move" thing CMake throws at you, I already built a thing into `setup.py` to check for this and clean automatically to avoid that silly and confusing error.